package.json

```diff
@@ -11,8 +11,8 @@
     "tslint": "tslint -p tsconfig.json --format stylish",
     "eslint": "eslint --cache '**/*.ts?(x)'",
     "build": "node ./node_modules/typescript/bin/tsc -b .",
-    "test": "uvu -r ts-node/register --ignore dist",
-    "update-snapshots": "uvu -r ts-node/register --ignore dist --update-snapshots",
+    "test": "TS_NODE_TRANSPILE_ONLY=true uvu -r ts-node/register --ignore dist",
```

Comment:
Sounds good! To clarify the reasoning behind this: while iterating on changes locally, I often have type errors because I want to test some logic but don't want to invest time into making Typescript happy about it because there's no intent to commit it. Type-checking slows down the process of trying different ideas on existing tests.

Reply:
I'm +1 to adding these scripts to package.json as long as we keep the typechecking variants as well. For sg/sg, I appreciate that `sg start` doesn't typecheck because it's a big codebase slowing down the edit/reload/debug feedback loop. In scip-typescript, typechecking has been fast enough for my needs.